.travis.yml

```diff
@@ -13,12 +13,25 @@ cache:
 addons:
   apt:
     packages:
-      - oracle-java8-installer
-      - oracle-java8-unlimited-jce-policy
+      #- oracle-java8-installer
+      #- oracle-java8-unlimited-jce-policy
+services:
+  - mysql
+before_install:
+  - mysql --version
+  - wget https://repo.mysql.com//mysql-apt-config_0.8.10-1_all.deb
+  - sudo dpkg -i mysql-apt-config_0.8.10-1_all.deb
+  - sudo apt-get update -q
+  - sudo apt-get install -q -y --allow-unauthenticated -o Dpkg::Options::=--force-confnew mysql-server
+  - sudo systemctl restart mysql
+  - sudo mysql_upgrade
+  - mysql --version
+  - mysql -e 'CREATE DATABASE AccountMetadata;'
+  - mysql -e 'USE AccountMetadata; SOURCE ./ambry-account/src/main/resources/AccountSchema.ddl;'
```

Comment:
One optimization I just thought of, now that we are splitting unit and int tests into separate jobs, is that only the int test job needs mysql.  Can this step be done on per-job basis?  (Can be follow up PR.)

Reply:
I think that may be possible, although I can't find documentation that explicitly says that you can change the services. However, I did find instructions for changing other parameters per each job: https://docs.travis-ci.com/user/build-matrix/#using-different-programming-languages-per-job

So, maybe moving `services` and `before_install` to the int test job will work:
```
jobs:
  include:
    - stage: test
      name: "Unit Tests"
      script: ./travis-unit-test.sh
    - stage: test
      name: "Integration Tests"
      script: ./travis-int-test.sh
      services:
        - mysql
      before_install:
        ...
    - stage: publish
      name: "Publish To Bintray"
      script: ./travis-publish.sh
```